Gds/src/fprime_gds/common/parsers/seq_file_parser.py

```diff
@@ -6,13 +6,13 @@
 
 
 class SeqFileParser:
-    def parse(self, filename):
+    @staticmethod
+    def parse(filename):
         """
         Generator that parses an input sequence file and returns a tuple
         for each valid line of the sequence file.
-        @param seqfile: A sequence file name (usually a .seq extension)
-        @return A list of tuples:
-            (lineNumber, descriptor, seconds, useconds, mnemonic, arguments)
+        :param filename: TODO: the parameter description should be filled by maintainer
```

Comment:
seqfile was most likely replaced by filename. The description should be the same
`A sequence file name (usually a .seq extension)`